pkg/projectfile/projectfile.go

```diff
@@ -816,8 +816,13 @@ func createCustom(params *CreateParams) (*Project, *failures.Failure) {
 	if runtime.GOOS == "windows" {
 		shell = "batch"
 	}
+
+	yaml := "sample_yaml"
+	if strings.ToLower(params.Language) == constants.PerlLanguageName {
```

Comment:
Please see if you can use http://github.com/ActiveState/cli/blob/77e2701270db9b9789e00e1274af701a5fa1443c/internal/language/language.go#L22-L22